sda-svc/templates/sftp-inbox-deploy.yaml

```diff
@@ -71,6 +71,8 @@ spec:
               key: mqPassword
         - name: BROKER_PORT
           value: {{ .Values.global.broker.port | quote }}
```

Comment:
i think we should add  a var for broker exchange, to be consistent with rest of services and we should add it to `values.yaml`
```
- name: BROKER_EXCHANGE
  value: {{ default "sda" .Values.inbox.brokerExchange }}
```

Reply:
`inbox.brokerExchange` is obsolete as all services talk to the same exchange now.

Reply:
But it should be fixed in the s3-proxy code.